samples/unit-tests/tooltip/shared/demo.js

```diff
@@ -1,3 +1,104 @@
+function generateData(n, k) {
```

Comment:
The `generateData` function and the tests that use it must be put inside an IIFE to avoid polluting the global namespace.